webui/react/src/hooks/useModal/Project/useModalProjectCreate.tsx

```diff
@@ -36,15 +47,18 @@ const useModalProjectCreate = ({ onClose, workspaceId }: Props): ModalHooks => {
       <div className={css.base}>
         <div>
           <label className={css.label} htmlFor="name">Name</label>
-          <Input id="name" maxLength={80} value={name} onChange={handleNameInput} />
+          <Input id="name" maxLength={80} onChange={handleNameInput} />
```

Comment:
**suggestion:** this is a required field I believe? A good form pattern to follow (using `Form.Item`) can be `useModalPasswordChange.tsx`

Reply:
it is actually checked [here](https://github.com/determined-ai/determined/pull/4691/files#diff-565663c0bf42a622416ed00decc158f8f7074bd7e4f19c84f95d85f823a650daR83), but I'll consider using it

Reply:
In this case, I think using Form is not appropriate because we handle modal with `getModalProps`